php-cs-test/README.md

```diff
@@ -0,0 +1,11 @@
+# Introduction
+
+The aim of this container image is to eliminate the need to `php-cs-fixer` in the composer requirements for each project we do. This image can be used on any environment that has Docker installed including a CI pipeline.
+
+# Usage
+
+Mount code to `/sut` on the container and we will test again prooph's rules.
```

Comment:
Has `sut` a special meaning? In other containers (f.e. prooph/php-cli) we mount $(pwd) to `/app`
Would it be ok to do the same here?